src/OneExplorer/OneExplorer.ts

```diff
@@ -206,8 +206,7 @@ class NodeFactory {
     //
     // A 'TreeDataProvider<element>' expects every elements (Node) to be correspond to visible
     // TreeItem, so let's not build hidden nodes.
-    if(attr && attr?.canHide === true && OneTreeDataProvider.didHideExtra === true)
-    {
+    if (attr && attr?.canHide === true && OneTreeDataProvider.didHideExtra === true) {
```

Comment:
Q) can we use `if (attr && attr?.canHide && OneTreeDataProvider.didHideExtra) {` without `=== true)` ?

Reply:
Yes :-D I just explicitly wrote it down as those fields can be 'undefined' or 'false'(either of them is false), but I think it's also clear with the shortened form.